src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -11,6 +11,7 @@
     height: 20px;
     width: 20px;
     background-image: url(/icons/nav-link-search-icon.png);
+    filter: brightness(0);
```

Comment:
It turns the icon black(since it is loaded as a file?/ not an icon). Didn't want to change the logic of this icon